common/activity.mk

```
@@ -0,0 +1,23 @@
+INTERNAL_PORT := 80
+CONT_NAME := ${IMG_NAME}
+FILE_FLAG := ../flag
+FLAG := $(shell cat $(FILE_FLAG))
```

```suggestion
FLAG := $(cat $(FILE_FLAG))
```
Is this necessary? On Mac for example it fails because of this 😄

---

AFAIK, it's mandatory, I tried without and it doesn't work.

@razvand , king of Makefiles, any idea how to make this working also on Mac?

---

I tried without it and it doesn't work on Ubuntu / Debian.
I would keep like these since the VMs where the challenges are deployed are Debian VMs.